.github/workflows/continuous-integration.yml

```diff
@@ -61,8 +61,8 @@ jobs:
     - name: 'Coverage'
       run: |
         ./gradlew :jacocoSelektTestReport ${{ env.SCAN }}
-        bash scripts/codecov
```

Comment:
The coverage report hasn't appeared on the PR as yet, I wonder if it's because codecov will now upload after Kover runs and this might cause codecov confusion. A quick way to verify this is to comment out the `./gradlew :koverHtmlReport ${{ env.SCAN }}` line and (just in case) the archiving `build/reports/kover/html` line too.

Reply:
I updated it slightly to maintain the previous ordering - hopefully this should address it